metabase/metabase
# fix narrow ordinal line charts have no tooltip

Closes https://github.com/metabase/metabase/issues/47847

### Description

Fixes line charts with ordinal x-axis when narrow do not have tooltips. This is because by default ECharts hide circles on narrow categorical line charts and tooltips are shown by hovering on a circle. This [setting](https://echarts.apache.org/en/option.html#series-line.showAllSymbol) enables always rendering all symbols.

### How to verify

- New Orders: Count by Created At (Day)
- Change x-axis type to Ordinal
- Make it narrow
- Ensure hovering on the line shows the tooltip

### Checklist
